gogettr/capabilities/registration_status.py

```diff
@@ -0,0 +1,18 @@
+from gogettr.capabilities.base import Capability
+
+
+class RegistrationStatus(Capability):
+    def is_registered(
```

Comment:
The function name here generally should be `pull(...)`, overriding the `pull(...)` from `Capability`.